pkg/daemon/ceph/osd/volume.go

```diff
@@ -189,7 +189,7 @@ func (a *OsdAgent) configureCVDevices(context *clusterd.Context, devices *Device
 		}
 		osds = append(osds, rawOsds...)
```

Comment:
Instead of trimming duplicates, why not skip appending them to this list if they already exist? At least it seems simpler to avoid adding a duplicate instead of removing it later after already added.

Reply:
Actually, why does raw mode even detect the LVM OSDs? Seems like the raw mode shouldn't detect them.

Reply:
> Instead of trimming duplicates, why not skip appending them to this list if they already exist? At least it seems simpler to avoid adding a duplicate instead of removing it later after already added.

Yes, it's easier, changed the code in that regard.

Reply:
> Actually, why does raw mode even detect the LVM OSDs? Seems like the raw mode shouldn't detect them.

RAW detects LVs because they are block devices, it's possible for someone to use RAW mode on an LV if desired.